alibi_detect/cd/base.py

```diff
@@ -462,7 +462,7 @@ def __init__(
             preprocess_x_ref: bool = True,
             update_x_ref: Optional[Dict[str, int]] = None,
             preprocess_fn: Optional[Callable] = None,
-            sigma: Optional[np.ndarray] = None,
+            # sigma: Optional[np.ndarray] = None,
```

Comment:
General remark - commented out code should be removed before the PR is final. Is there a particular reason to have these lines commented out during review rather than removed?